spring-cloud-zuul-ratelimit-core/src/main/java/com/marcosbarbero/cloud/autoconfigure/zuul/ratelimit/config/properties/RateLimitProperties.java

```diff
@@ -72,19 +62,41 @@
     @NotNull
     @Value("${spring.application.name:rate-limit-application}")
     private String keyPrefix;
-    @Valid
     @NotNull
     private RateLimitRepository repository;
     private int postFilterOrder = SEND_RESPONSE_FILTER_ORDER - 10;
     private int preFilterOrder = FORM_BODY_WRAPPER_FILTER_ORDER;
 
+    @Deprecated
```

Comment:
I think we can just remove the support for them.
They are deprecated for a while.